index.js

```diff
@@ -56,6 +61,11 @@ module.exports = function createPlugin(app) {
           path: 'networking.lte.radioQuality',
           value: radioQuality,
         });
+        const modemTemperature = Buffer.concat(data.slice(4, 7)).readInt32BE() / 10;
+        values.push({
+          path: 'networking.modem.temperature',
```

Comment:
Temperatures in Signal K should be in Kelvin, and are typically under `environment`. Can you change that?

Reply:
Changed to Kelvin and added meta data too.
Could we keep this under `networking.modem.temperature`, then all modem data is nicely grouped in Data Browser?

Reply:
Thanks! We're currently under way. I'll give it another look, and hopefully merge and release when we're in harbour again